preprod.healdata.org/metadata/aggregate_config.json

```diff
@@ -18,7 +18,7 @@
             }
         },
         "SAMHDA": {
-            "mds_url": "https://preprod.healdata.org",
+            "mds_url": "https://preprod.healdata.org/",
             "commons_url": "preprod.healdata.org",
```

Comment:
coress-checking with gitops-qa, do we need to update this with full url format?

Reply:
False alarm, I just confirmed this isn't necessary. Only the gen3 "adapter" commons need this